### PR TITLE
Added RUN_TYPE in the environment to send as metadata

### PR DESCRIPTION
**Pull request by tisutisu**
This PR will address enhancement being discussed in the story: https://issues.redhat.com/browse/APPSVC-1261

### CODE REVIEW

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml [hunk 1]
```diff
@@ -36,8 +36,10 @@ tests:
       commands: |
         if [ "$(echo -e $CLONEREFS_OPTIONS | jq -r '.refs[] | select(.org == "openshift").repo')" == "release" ]; then
           export OPERATOR_INDEX_IMAGE_REF=quay.io/redhat-developer/servicebinding-operator:index;
+          export RUN_TYPE='scratch';
         else
           export OPERATOR_INDEX_IMAGE_REF=quay.io/redhat-developer/servicebinding-operator:pr-${PULL_NUMBER}-${PULL_PULL_SHA:0:8}-index;
+          export export RUN_TYPE='pr-check';
```
Comment (pmacik) [inline ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml:42]: @tisutisu The job is missing the part for uploading the KPI to opensearch

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-release-v1.3.x.yaml [hunk 2]
```diff
@@ -36,8 +36,10 @@ tests:
       commands: |
         if [ "$(echo -e $CLONEREFS_OPTIONS | jq -r '.refs[] | select(.org == "openshift").repo')" == "release" ]; then
           export OPERATOR_INDEX_IMAGE_REF=quay.io/redhat-developer/servicebinding-operator:index-v1.3.x;
+          export RUN_TYPE='scratch';
         else
           export OPERATOR_INDEX_IMAGE_REF=quay.io/redhat-developer/servicebinding-operator:pr-${PULL_NUMBER}-${PULL_PULL_SHA:0:8}-index;
+          export RUN_TYPE='pr-check';
```
Comment (pmacik) [inline ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-release-v1.3.x.yaml:42]: @tisutisu The job is missing the part for uploading the KPI to opensearch

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml (hunk 1 above)
Comment (pmacik) [inline ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml:42]:
```suggestion
          export RUN_TYPE='pr-check';
```

**Review by pmacik — CHANGES_REQUESTED** [review]

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml (hunk 1 above)
Comment (tisutisu) [reply ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml]: My Bad. Added the uploading data part now.

Comment (tisutisu) [reply ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml]: Fixed with the latest commit.

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-release-v1.3.x.yaml (hunk 2 above)
Comment (tisutisu) [reply ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-release-v1.3.x.yaml:42]: My Bad. Added the uploading data part now.

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml [hunk 3]
```diff
@@ -50,6 +52,7 @@ tests:
         source /var/run/secrets/sbo-on-sb-perf/registry-credentials-env.sh
```
Comment (pmacik) [inline ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml:52]:
```suggestion
        source /var/run/secrets/sbo-on-sb-perf/registry-credentials-env.sh
        source /var/run/secrets/sbo-on-sb-perf/opensearch-aws-credentials-env.sh
```

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-release-v1.3.x.yaml [hunk 4]
```diff
@@ -50,6 +52,7 @@ tests:
         source /var/run/secrets/sbo-on-sb-perf/registry-credentials-env.sh
```
Comment (pmacik) [inline ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-release-v1.3.x.yaml:52]:
```suggestion
        source /var/run/secrets/sbo-on-sb-perf/registry-credentials-env.sh
        source /var/run/secrets/sbo-on-sb-perf/opensearch-aws-credentials-env.sh
```

**Review by pmacik — CHANGES_REQUESTED** [review]
OpenSearch credentials are also missing

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-release-v1.3.x.yaml (hunk 4 above)
Comment (tisutisu) [reply ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-release-v1.3.x.yaml:52]: Added with the latest commit. Thanks.

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml (hunk 3 above)
Comment (tisutisu) [reply ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml:52]: Added with the latest commit. Thanks.

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.11.yaml [hunk 5]
```diff
@@ -116,6 +116,7 @@ tests:
         export OPENSHIFT_API=$(yq e '.clusters[0].cluster.server' $KUBECONFIG)
         export OPENSHIFT_USERNAME=kubeadmin
         export OPENSHIFT_PASSWORD=$(cat $KUBEADMIN_PASSWORD_FILE)
+        export RUN_TYPE='latest-openshift'
```
Comment (pmacik) [inline ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.11.yaml:119]:
```suggestion
        if [ "$(echo -e $CLONEREFS_OPTIONS | jq -r '.refs[] | select(.org == "openshift").repo')" == "release" ]; then
          export RUN_TYPE='scratch';
        else
          export RUN_TYPE='latest-openshift';
        fi
```

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.12.yaml [hunk 6]
```diff
@@ -94,6 +94,7 @@ tests:
         export OPENSHIFT_API=$(yq e '.clusters[0].cluster.server' $KUBECONFIG)
         export OPENSHIFT_USERNAME=kubeadmin
         export OPENSHIFT_PASSWORD=$(cat $KUBEADMIN_PASSWORD_FILE)
+        export RUN_TYPE='next-openshift'
```
Comment (pmacik) [inline ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master__4.12.yaml:97]:
```suggestion
        if [ "$(echo -e $CLONEREFS_OPTIONS | jq -r '.refs[] | select(.org == "openshift").repo')" == "release" ]; then
          export RUN_TYPE='scratch';
        else
          export RUN_TYPE='next-openshift';
        fi
```

File: ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml [hunk 7]
```diff
@@ -78,6 +82,7 @@ tests:
         export OPENSHIFT_API=$(yq e '.clusters[0].cluster.server' $KUBECONFIG)
         export OPENSHIFT_USERNAME=kubeadmin
         export OPENSHIFT_PASSWORD=$(cat $KUBEADMIN_PASSWORD_FILE)
+        export RUN_TYPE='current-dev-sandbox'
```
Comment (pmacik) [inline ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml:85]:
```suggestion
        if [ "$(echo -e $CLONEREFS_OPTIONS | jq -r '.refs[] | select(.org == "openshift").repo')" == "release" ]; then
          export RUN_TYPE='scratch';
        else
          export RUN_TYPE='current-dev-sandbox';
        fi
```

**Review by pmacik — CHANGES_REQUESTED** [review]
We need to mark the `run_type` as `scratch` also for the weekly jobs.

**Comment by pmacik** [comment]
@tisutisu I have Incorporated theses changes into https://github.com/openshift/release/pull/35524

**Comment by tisutisu** [comment]
Closing this PR, since changes has been incorporated into https://github.com/openshift/release/pull/35524

**Pull request closed without merging**